README.md

```diff
@@ -49,6 +49,14 @@ QuestDB release and sample datasets:
 - live trading data from a cryptocurrency exchange
```

Comment:
Maybe we can give an indication or size, or mention it has data from March 2022 onwards and each month has between 32M and 45M rows

Reply:
I'd like to keep it shorter, how about:

-  live cryptocurrency market data with 30M+ rows per month

Reply:
Works for me